topography/include/gaussian_topog.inc

```diff
@@ -29,72 +29,36 @@
 
 !> @addtogroup gaussian_topog_mod
 !> @{
-module gaussian_topog_mod
 
-use  fms_mod, only: check_nml_error,                 &
-                    stdlog, write_version_number,    &
-                    mpp_pe, mpp_root_pe,             &
-                    error_mesg, FATAL
-
-use constants_mod, only: pi
-
-use mpp_mod,       only: input_nml_file
-
-implicit none
-private
-
-public :: gaussian_topog_init, get_gaussian_topog
+!#######################################################################
 
-!-----------------------------------------------------------------------
-! <NAMELIST NAME="gaussian_topog_nml">
-!   <DATA NAME="height" UNITS="meter" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     Height in meters of the Gaussian mountains.
-!    </DATA>
-!   <DATA NAME="olon, olat" UNITS="degree" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     The longitude and latitude of mountain origins (in degrees).
-!    </DATA>
-!   <DATA NAME="wlon, wlat" UNITS="degree" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     The longitude and latitude half-width of mountain tails (in degrees).
-!    </DATA>
-!   <DATA NAME="rlon, rlat" UNITS="degree" TYPE="real" DIM="(mxmtns)" DEFAULT="0.">
-!     The longitude and latitude half-width of mountain ridges (in degrees).  For a
-!     "standard" Gaussian mountain set rlon=rlat=0.
-!    </DATA>
-!
-!    <DATA NAME="NOTE">
-!     The variables in this namelist are only used when routine
-!     <TT>gaussian_topog_init</TT> is called.  The namelist variables
-!     are dimensioned (by 10), so that multiple mountains can be generated.
+!> @brief Returns a simple surface height field that consists of a single
```

Comment:
can you move this comment block to right above the function its documenting (get_gaussian_topog). If its not right above the routine it won't get parsed correctly.

Reply:
fixed in b55e5dad

Reply:
You only moved the first 3 lines (the brief description), the rest ofit needs to be moved (lines 35-59)

Reply:
@mcallic2 could you make these changes?

Reply:
I'm still lost as to why the comments for get_gaussian_topog is moved here above gaussian_topog_init...

Reply:
Do lines 34-35 also need to be moved down?

Reply:
I think they are fine, it'll just read the first !> as the brief descriptions and the second part as the expanded.